Source/WebKit/WebProcess/Plugins/PDF/WKAccessibilityPDFDocumentObject.mm

```diff
@@ -297,6 +360,9 @@ - (void)gotoDestination:(PDFDestination *)destination
         plugin->accessibilityScrollToPage(pageIndex);
     });
 }
+
+#endif
```

Comment:
Seems like we don't follow this convention elsewhere in this file, but I find it nice to include the compiler flag associated with this `#endif` in a comment like so:

```cpp
#endif // PLATFORM(FOO)
```